experimental/dds/tree2/src/test/shared-tree/schematizeTree.spec.ts

```diff
@@ -88,4 +157,38 @@ describe("schematizeView", () => {
 		// Initial tree should not be applied
 		assert.equal(schematized.root, undefined);
 	});
+
+	it("errors if schema changes to not be compatible with view schema", () => {
+		const provider = new TestTreeProviderLite(2, factory);
+		const tree = provider.trees[0];
+		const tree2 = provider.trees[1];
+
+		const treeLog = [];
```

Comment:
You're not using `treeLog`. I have nothing against keep it here anyway, but since it is here, you may want to use it to make test expectations more easily understandable at a glance.

Reply:
I was using for debugging, I had intended to remove it. I guess I missed a spot